### PR TITLE
Feature/wrf columbia t2

### PR DESCRIPTION
**Pull request by timbbaldwin**
Ready to be reviewed/merged into develop.

Added Processor for T2 (temperature).

Updated wrf-columbia-precip in product table dss_fpart.

Added unit for DEG F.

### CODE REVIEW

File: database/sql/04-products.sql [hunk 1]
```diff
@@ -149,6 +149,7 @@ INSERT INTO acquirable (id, name, slug) VALUES
     ('5c0f1cfa-bcf8-4587-9513-88cb197ec863', 'ndfd_conus_airtemp', 'ndfd-conus-airtemp'),
     ('d4e67bee-2320-4281-b6ef-a040cdeafeb8', 'hrrr_total_precip','hrrr-total-precip'),
     ('ec926de8-6872-4d2b-b7ce-6002221babcd', 'wrf_columbia_precip','wrf-columbia-precip'),
+    ('552bf762-449f-4983-bbdc-9d89daada260', 'wrf_columbia_t2','wrf-columbia-t2'),
```
Comment (adamscarberry) [inline database/sql/04-products.sql]: suggest calling the slug **wrf-columbia-airtemp**

File: database/sql/04-products.sql [hunk 2]
```diff
@@ -190,7 +191,8 @@ INSERT INTO product (id, name, slug, temporal_duration, temporal_resolution, dss
...
-    ('b50f29f4-547b-4371-9365-60d44eef412e','wrf_columbia_precip','wrf-columbia-precip',3600,3600,'WRF-COLUMBIA','eb82d661-afe6-436a-b0df-2ab0b478a1af','e245d39f-3209-4e58-bfb7-4eae94b3f8dd', 'WRF Columbia precipitation data created for the entire Columbia River Basin'),
+    ('b50f29f4-547b-4371-9365-60d44eef412e','wrf_columbia_precip','wrf-columbia-precip',3600,3600,'WRF-COLUMBIA-PRECIP','eb82d661-afe6-436a-b0df-2ab0b478a1af','e245d39f-3209-4e58-bfb7-4eae94b3f8dd', 'WRF Columbia precipitation data created for the entire Columbia River Basin'),
```
Comment (adamscarberry) [inline database/sql/04-products.sql]: DSS F part doesn't need the parameter as that will be automatically added to the C Part of a DSS path.  I would keep it **WRF-COLUMBIA**

File: database/sql/04-products.sql [hunk 3]
```diff
@@ -190,7 +191,8 @@ INSERT INTO product (id, name, slug, temporal_duration, temporal_resolution, dss
...
+    ('793e285f-333b-41a3-b4ab-223a7a764668','wrf_columbia_t2','wrf-columbia-t2',3600,3600,'WRF-COLUMBIA-T2','5fab39b9-90ba-482a-8156-d863ad7c45ad','0c8dcd1f-93db-4e64-be1d-47b3462deb2a', 'WRF Columbia T2 (temperature at 2 m) data created for the entire Columbia River Basin'),
```
Comment (adamscarberry) [inline database/sql/04-products.sql]: I would call the slug **wrf-columbia-airtemp** and DSS F Part **WRF-COLUMBIA**

**Review by adamscarberry — CHANGES_REQUESTED** [review]

File: database/sql/04-products.sql (hunk 1 above)
Comment (timbbaldwin) [reply database/sql/04-products.sql]: Changed

File: database/sql/04-products.sql (hunk 2 above)
Comment (timbbaldwin) [reply database/sql/04-products.sql]: Changed

File: database/sql/04-products.sql (hunk 3 above)
Comment (timbbaldwin) [reply database/sql/04-products.sql]: Changed

**Review by adamscarberry — APPROVED** [review]

**Pull request merged**